### ISSUE TITLE
Test IAM Permissions generated code failing

### ISSUE THREAD

**Issue by kolea2** [googleapis/gapic-generator-python#778]
In a recent regen for Bigtable, it added: 

https://github.com/googleapis/python-bigtable/pull/216/files#diff-9bbb5ea4e132c72b65ab1881c88e2ab4580a237b5bd8e4f4e8709a68a564f79cR2050-R2052

The generated [test](https://github.com/googleapis/python-bigtable/blob/master/tests/unit/gapic/bigtable_admin_v2/test_bigtable_instance_admin.py#L4456) is now failing with the error, 
```
E           AssertionError: assert ['permissions...ssions_value'] == ['permissions_value']
E             Left contains one more item: 'permissions_value'
E             Full diff:
E             - ['permissions_value']
E             + ['permissions_value', 'permissions_value']
```

I believe it is re-appending the permissions value incorrectly (the permissions are already set [here](https://github.com/googleapis/python-bigtable/pull/216/files#diff-9bbb5ea4e132c72b65ab1881c88e2ab4580a237b5bd8e4f4e8709a68a564f79cR2044)). Please let me know if you need any more info. Thanks for the help!

Full PR and diff: https://github.com/googleapis/python-bigtable/pull/216

**Comment by kolea2** [comment]
Similar to: https://github.com/googleapis/gapic-generator-python/issues/756

**Comment by crwilcox** [comment]
Checking in. Any progress on this by chance?

**Comment by busunkim96** [comment]
I also observed this on `google/cloud/tasks`, so I suspect it is widespread (probably any API with `test_iam_permissions`?). This replace was enough to get the unit tests passing again:

```py
s.replace(
    "google/cloud/**/client.py",
    """\s+if permissions:
\s+request\.permissions\.extend\(permissions\)""",
    "",
)
```

**Comment by software-dov** [comment]
Fixed by #786
